.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
+end_of_line = lf

```

Comment:
That shouldn't be there IMO. End of line should either be left to the user (git has a checkout config to deal with that) or on a per file extension basis (using `.gitattributes` file).

Each system (Windows, Linux, Mac) has its own convention and tools/editor/CLI work better with the native line endings.

Reply:
Removed